netsetup/__main__.py

```diff
@@ -17,6 +17,7 @@
 import gobject as GObject
 
 import daemon
+from .desktop_bus import DesktopBus
```

Comment:
I would prefer DbusService.
What do you think?

Reply:
It's better